packages/fluentui/react-northstar/src/themes/teams/components/Datepicker/datepickerCalendarCellStyles.ts

```diff
@@ -1,63 +1,42 @@
 import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
 import { DatepickerVariables } from './datepickerVariables';
 import { DatepickerCalendarCellStylesProps } from '../../../../components/Datepicker/DatepickerCalendarCell';
-import { getBorderFocusStyles } from '../../getBorderFocusStyles';
+
+const cellStyles = (p: DatepickerCalendarCellStylesProps, v: DatepickerVariables) => ({
+  ...(p.quiet && {
+    color: v.calendarCellQuietColor,
+  }),
+
+  ...(p.selected && {
+    color: v.calendarCellSelectedColor,
+    backgroundColor: v.calendarCellSelectedBackgroundColor,
+  }),
+
+  ...(!p.disabled && {
```

Comment:
Let's avoid negative conditions in styles

Reply:
I see.

What is then the preferred coding style for creating styling properties in case a given boolean is true and in case not?

Reply:
Have always a default value and then override it with a positive condition.

Reply:
Updated.